README.md

```diff
@@ -13,8 +13,15 @@ providing additional resources, or more generally improving the content.
 
 ## Running locally
 
-Requires [cobalt](https://cobalt-org.github.io/) `>=0.17.5`
-Recommend [just](https://github.com/casey/just) as a task runner (or using the commands in the [Justfile](Justfile))
+- Requires [cobalt](https://cobalt-org.github.io/) `>=0.17.5`
+    - The most recent official release is `0.17.4`; installation from the main git repo using
```

Comment:
Looks like we can already remove this note and start using `0.17.5`: https://github.com/cobalt-org/cobalt.rs/pull/941#issuecomment-990216360

Reply:
Sounds good, I've removed that part of the README!